iceoryx_utils/include/iceoryx_utils/posix_wrapper/semaphore.hpp

```diff
@@ -185,8 +186,7 @@ class Semaphore : public DesignPattern::Creation<Semaphore, SemaphoreError>
     iox_sem_t* getHandle() noexcept;
 
   private:
-    static constexpr int m_nameSize = 128;
-    char m_name[m_nameSize] = {'\0'};
+    cxx::string<128> m_name;
```

Comment:
I guess that'll be cleaned-up when a common convention for string aliases is found in #260?!

Reply:
yes, the idea is to have cxx::strings in the classes itself and for the last PR of #260 do a grep for all string aliases or direct usages and consolidate it. We could also decide to refactor the Semaphore and use cxx::string in the API instead of a `const char*`, but that is a bigger refactoring topic and maybe something for v1.1